File: BinaryTree_Inorder_Traversal.java
class Solution {
    public List<Integer> inHelper(TreeNode root,List<Integer>list)
    {
        if(root==null)
        {
            return list ;
        }
        inHelper(root.left,list);
        list.add(root.val);
        inHelper(root.right,list);
        return list;
    }
    public List<Integer> inorderTraversal(TreeNode root) {
        List<Integer>list=new ArrayList<>();
        return inHelper(root,list);
        }
}

File: Binary Tree Inorder Traversal.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right
class Solution:
    def inorderTraversal(self, root: Optional[TreeNode]) -> List[int]:
        l=[]
        def ino(node) :
            if not node :
                return 
            ino(node.left)
            l.append(node.val)
            ino(node.right)
        ino(root)
        return l
